mart/configs/attack/pgd.yaml

```diff
@@ -0,0 +1,7 @@
+defaults:
+  - perturber: default
+  - perturber/initializer: uniform
+  - /optimizer@optimizer: sgd
+
+optimizer:
+  lr: ${..lr}
```

Comment:
How do you know this exists if this doesn't have `adversary.yaml` as default?

Reply:
You are right. Now I have moved `lr: ???` to `pgd.yaml`.